config/sync/filter.format.underlined_title.yml

```diff
@@ -0,0 +1,17 @@
+uuid: c5de36b9-4fad-49ec-99ad-d8ca1404c989
+langcode: en
+status: true
+dependencies: {  }
+name: 'Underlined title'
+format: underlined_title
+weight: 0
+filters:
+  filter_html:
+    id: filter_html
+    provider: filter
+    status: true
+    weight: -10
+    settings:
+      allowed_html: '<br> <p> <u>'
```

Comment:
Please remove `<p>` from the allowed html.

[`<p>` tags are not allowed within headings](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#technical_summary).

Reply:
I checked this out and as far as I can tell this is not actually possible. I see that we have not enabled converting line breaks to `<p>` and `<br>` so that is probably sufficient.

Reply:
Yeah i dont see a way to do this either. 

![image](https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/13272656/dee6e78e-b141-40e5-93fe-905d15c227bd)